blivet/osinstall.py

```diff
@@ -1188,13 +1276,13 @@ def do_it(self, callbacks=None):
         self.dump_state("final")
 
     def write(self):
-        Blivet.write(self)
+        super().write()
 
         self.make_mtab()
         self.fsset.write()
-        iscsi.write(get_sysroot(), self)
-        fcoe.write(get_sysroot())
-        zfcp.write(get_sysroot())
+        iscsi.write(self.sysroot, self)
```

Comment:
Eh, you technically can but in blivet.Blivet I defined the property "sysroot" (which is r/w). I'd say for the sake of being proper it should be self.sysroot.